### DISCUSSION TITLE
Misregistration issues with ERS-2 SLCs in stripmapApp

### DISCUSSION THREAD

**Discussion by fdelgadodelapuente** [isce-framework/isce2#473] [Q&A]
Dear ISCE developers,

I've tried to process several ERS-2 raw images in the CEOS file format with ODR orbits with stripmapApp, but the software usually fails to align the SLCs. The coarse and refined secondary SLC tend to have a worst alignment than the non-coregistered secondary SLC. However, I can process them with no problems with the deprecated insarApp. I wonder if the problem is due to the low accuracy of the ODR state vectors for the geometric coregistration or some other issue?

Thanks a lot
Francisco

**Comment by piyushrpt** [comment]
It is very likely due to azimuth time tag errors being large in ERS-2. Orbits are fine but time tags on the image can be off. Would always recommend the on-demand SLCs from ESA over using RAW data, as a lot of these issues are addressed in generation of those SLCs.

**Reply by fdelgadodelapuente** [reply]
Thanks Piyush for the explanation. Probably a lot of the old ERS data in SSARA likely have these issues.

Francisco

**Reply by piyushrpt** [reply]
They likely came from being downloaded at different ground stations but not sure.

**Reply by fdelgadodelapuente** [reply]
Dear ISCE developers,

I've also noticed that the error also shows up when processing several ERS-1/2 tandem data in both Patagonia and Alaska, but using ESA SLCs with stripmapApp instead of the old CEOS raw data -- ISCE is unable to coregister them. Is it due to the same issue with the azimuth time tag errors?

Francisco

**Reply by piyushrpt** [reply]
Very likely so. However, I haven't seen many failures with the on-demand ESA SLCs.
